_config.yml

```diff
@@ -174,6 +174,22 @@ archive_pagination: true
 category_pagination: true
 tag_pagination: true
 
+# Gitment - A github issue comment tool.
+# Repository: https://github.com/imsun/gitment
```

Comment:
Just put `Read https://github.com/imsun/gitment#get-started to setup Gitment`.

It will be easier to maintain (less links)